src/main/java/org/jabref/gui/collab/ChangeScanner.java

```diff
@@ -39,7 +38,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ChangeScanner.class);
 
-    private final File file;
+    private final Path file;
```

Comment:
Please make this an optional (if it really makes sense to have a null value here...which I kind of doubt).

Reply:
I already considered this, I think it could happen if you have a shared database which exists only i memory